### ISSUE TITLE
Files opened via (markdown) link code blank

### ISSUE THREAD

**Issue by Blackhawke** [gsantner/markor#1376]
Linked files (using markdown code [foo](foo.md) are blank when clicked on in view mode.

I've moved to Markor from a different editor and we have existing index files _full_ of links to other files in our private git repo. (It's basically an internal wiki.) The paths, as written were your typical (./foo.md) and worked fine. I've tried changing them to complete, absolute paths, and I've tried using Markor's automatic link generator, just to see if it wants something different.

Naadah! Same results. No file name on the top bar, and a blank file.

**Comment by gsantner** [comment]
add app version, as the template lets you know.

**Comment by Blackhawke** [comment]
Nice!

Anyway, my bad.

App Version 2.7.0 (123)
Running on 2 Galaxy Tab As, Android 10 and 11 (one recently upgraded, the other not)

And oh! I'm using the GitHub app, and saw no template.

**Comment by gsantner** [comment]
I cannot reproduce, works fine here.

![Screenshot_20210630-015737_Trebuchet](https://user-images.githubusercontent.com/6735650/123882300-cae6e600-d946-11eb-87d9-a55abbe984c6.png)



https://user-images.githubusercontent.com/6735650/123882269-b9054300-d946-11eb-949e-479261ffe1de.mp4

**Comment by Blackhawke** [comment]
That's interesting. Here's what it looks on my Galaxy Tab A (still running android 10) when I try to open a file from a (markdown) link in another file.

![Screenshot_20210630-092631_Markor](https://user-images.githubusercontent.com/1757829/123999589-89c2f600-d987-11eb-8ee5-47eafbb29890.jpg)

**Comment by gsantner** [comment]
> still running android 10

well thats a sane, recent Android version ;D. Though I cannot help you with any of my devices as I cannot reproduce.

**Comment by Blackhawke** [comment]
Argh!! You're the dev so I thought maybe you could say why it works in our other markdown editor/viewer but not this one. :/

Well... We're switching to Legacy OS (v17.1), because Android 11 BORKS git. Every push/pull is so slloooowwwwwww it appears we're running into Google's silly "SafeyNet". Anyway, maybe that'll fix it.

**Comment by Blackhawke** [comment]
Okay, I'm updating this bug, even though you guys have closed it, because I just changed tablets. As reported, it originally showed up on a Samsung Galaxy Tab A. Now I'm running a Samsung Galaxy S6 Lite (stock) and I get the exact same result: Opening file links in Markor from another file in Markor results in the blank Markor screen shown in the screenshot.

Also, I rooted my wife's Tab A. (the bug was reported when we were running Samsung's pre-installed rom.) It's now running a Nexus style offshoot of LegacyOS and it does the same thing. So that's 3 roms on 2 different devices, all producing the same bug. 

Maybe this will help you reproduce the bug.

**Comment by gsantner** [comment]
I use Markor everyday and didn't see it yet. Might be issue with your Android rom and it's webview / custom tab application. I suggest you to also try with installing Firefox Android, which also supports custom tabs. That might help.

**Comment by Blackhawke** [comment]
Well, let's see, if it's a problem with my rom, then there's a problem with every Samsung Galaxy Tab A and Samsung Galaxy S6 (lite) on the planet. Because, as I have said, they're stock. The Tab A was (when I first reported this bug) still running Samsung's Android 10. When it OTAed to Android 11 the problem didn't go away. The S6 is brand new, out of the box. It OTAed to their version of Android 11 before I even got to open Makor, and it's behaving the exact same way.

So that's two out of three use cases, and the devices are running warranteed software.

The third, as I explained, is a rooted, Nexus version of LegacyOS Android 10, now running on one of the Tab A's. Exact same behavior as the stock roms on both the Tab A and S6 (lite).

So finally, to quote you "I run Markor every day". Or, we *now* run Markor everyday. For the last four years we ran the paid version of Epsilon Notes. Then, a little over a year ago, the dev vanished. It appears the app has been abandoned, so we're trying out new markdown apps that look like they'll work with our doc structure. Save for this very annoying and elusive bug---and one other wierd but minor quirk---Markor looks like a good fit.

**Comment by Blackhawke** [comment]
**I found it!**

I found the bug. Our files are in a folder named (in the style of) ``foo+bar``. This has never been a problem (on our linux desktops and in other android editors for 10+ years). However, one thing I noticed and just wrote off as a quirk level bug in Markor is that, every once in awhile it would create a new path to the folder the files are in named ``foo bar``.

So, I just looked at the "info" for the blank file Markor is opening and, guess what. The path is _entirely_ correct save for one thing: It's looking in ``<root path>/foo bar/file.md``, which doesn't exist! The correct path is ``<root path>/foo+bar/file.md``.

Hope that helps, and maybe now you don't just think I'm a pain in the A***. :)
